src/Collections/Sequences/MergeSort.dfy

```diff
@@ -0,0 +1,70 @@
+ include "../../Relations/Relations.dfy"
+ 
+ /**********************************************************
+  *
+  *  Sorting of Sequences
+  *
+  ***********************************************************/
+
+module Seq.MergeSorting {
+
+    import opened Comparison = Relations.Comparison
+ 
+  predicate method SortedBy<T>(a: seq<T>,compare: (T, T) -> CompResult)
```

Comment:
This and the following lemma should be moved to `Comparison.dfy` since it's not specific to any particular sorting algorithm.